add more docs for response helpers (#958)
solidjs/solid-docs: src/routes/solid-router/reference/response-helpers/reload.mdx

---
title: reload
---

Reload is a response helper built on top of [revalidate](/solid-router/response-helpers/revalidate).

> It will receive the either a cache key, or an array of cache keys, invalidate those queries, and fire them again.

Reviewer: ```suggestion
It will receive a cache key, or an array of cache keys, to invalidate those queries, and cause them to fire again.
```
